aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -111,13 +111,6 @@ module aptos_framework::fungible_asset {
         frozen: bool,
     }
 
-    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
-    struct FungibleAssetEvents has key {
```

Comment:
This would violate the upgrade policy checks

Reply:
I put it at the bottom.